reference/config_files/profiles_default.rst

```diff
@@ -0,0 +1,484 @@
+    - Check the command :ref:`conan profile <reference_commands_profile>`.
+
+
+Profile rendering
+-----------------
+
+The profiles are rendered as **jinja2** templates by default. When Conan loads a profile, it immediately parses and
+renders the template, which must result in a standard text profile.
+
+Some of the capabilities of the profile templates are:
+
+- Using the platform information, like obtaining the current OS is possible because the
+  Python ``platform`` module is added to the render context:
+
+  .. code:: jinja
+
+     [settings]
+     os = {{ {"Darwin": "Macos"}.get(platform.system(), platform.system()) }}
+
+- Reading environment variables can be done because the Python ``os`` module is added
+  to the render context.:
+
+  .. code:: jinja
+
+     [settings]
+     build_type = {{ os.getenv("MY_BUILD_TYPE") }}
+
+- Defining your own variables and using them in the profile:
+
+  .. code:: jinja
+
+     {% set os = "FreeBSD" %}
+     {% set clang = "my/path/to/clang" %}
+
+     [settings]
+     os = {{ os }}
+
+     [conf]
+     tools.build:compiler_executables={'c': '{{ clang }}', 'cpp': '{{ clang + '++' }}' }
+
+
+- Joining and defining paths, including referencing the current profile directory. For
+  example, defining a toolchain which file is located besides the profile can be done.
+  Besides the ``os`` Python module, the variable ``profile_dir`` pointing to the current profile
+  folder is added to the context.
+
+  .. code:: jinja
+
+       [conf]
+       tools.cmake.cmaketoolchain:toolchain_file = {{ os.path.join(profile_dir, "toolchain.cmake") }}
+
+- Including or importing other files from ``profiles`` folder:
+
+  .. code-block:: jinja
+     :caption: profile_vars.jinja
+
+     {% set a = "Debug" %}
+
+  .. code-block:: jinja
+     :caption: profile1.jinja
+
+     {% import "profile_vars.jinja" as vars %}
+     [settings]
+     build_type = {{ vars.a }}
+
+- Any other feature supported by *jinja2* is possible: for loops, if-else, etc. This
+  would be useful to define custom per-package settings or options for multiple packages
+  in a large dependency graph.
+
+
+Profiles sections
+-----------------
+
+These are the available sections in profiles:
+
+[settings]
+++++++++++
+
+List of settings available from **settings.yml**:
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [settings]
+    arch=x86_64
+    build_type=Release
+    compiler=apple-clang
+    compiler.cppstd=gnu98
+    compiler.libcxx=libc++
+    compiler.version=12.0
+    os=Macos
+
+
+[options]
+++++++++++
+
+List of options available from your recipe and its dependencies:
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [options]
+    my_pkg_option=True
+    shared=True
+
+
+[tool_requires]
++++++++++++++++
+
+List of ``tool_requires`` required by your recipe or its dependencies:
+
+.. code-block:: text
+    :caption: *myprofile*
+
+    [tool_requires]
+    cmake/3.25.2
+    cmake/3.20.6
```

Comment:
Incompatible, can't have 2 versions of same tool_require

Reply:
bad example